utils/getManualToc.js

```diff
@@ -0,0 +1,36 @@
+function getManualToc(tree, pageUrl) {
+    if (!tree.children || !pageUrl) {
+        return []
+    }
+
+    let currentChildren = tree.children
+    let tableOfContentArrForSet = []
+    let currentPageChildren
+
+    for (const currentPageUrl of pageUrl) {
+        if (currentPageChildren) {
+            currentPageChildren =
+                currentPageChildren.find((obj) => obj.url === currentPageUrl)?.children || []
+        }
+
+        currentChildren = currentChildren?.find(
+            (obj) => obj?.properties?.pageUrl?.url === currentPageUrl
+        )?.children
+
+        const b = currentChildren?.map((obj) => ({
```

Comment:
Maybe rename to `manualSections` in this MR ?

Reply:
I don't mind. But this, it seems, is not `manualSections'.
Sometimes it's `manualSections' and sometimes it's `manualList`. We'll need to redo this later.

Reply:
I don't mind. But it's not `manualSections`.

Sometimes it's `manualList`. We'll need to redo this later.